File: 20140301/fizzbuzz.py
def to_str(num):
    if num % 15 == 0:
        return "FizzBuzz"
    elif num % 3 == 0:
        return "Fizz"
    elif num % 5 == 0:
        return "Buzz"
    else:
        return num

def fizzbuzz(min,max):
    for i in range(min,max):
        print(to_str(i))

fizzbuzz(1,100)

File: 20140721/fizzbuzz.js
function toStr(num) {
  if (num % 15 == 0)
    return "FizzBuzz";
  else if (num % 3 == 0)
    return "Fizz";
  else if (num % 5 == 0)
    return "Buzz";
  else
    return num;
}

function fizzbuzz(min, max) {
  for (var i = min; i <= max; i++) {
    print(toStr(i));
  }
}

fizzbuzz(1, 100);